app/src/main/java/np/com/naxa/factsnepal/surveys/SurveyActivity.java

```diff
@@ -166,31 +353,49 @@ private CardView convert(JSONObject jsonObject, Context context) {
             JsonView jsonView = new JsonView(context);
             jsonView.init(params).create();
 
-           return wrapByCardView(jsonView);
-        }catch (Exception e){e.printStackTrace();
-        return wrapByCardView(new LinearLayout(context));
+            return wrapByCardView(jsonView);
+        } catch (Exception e) {
+            e.printStackTrace();
+            return wrapByCardView(new LinearLayout(context));
         }
     }
 
     @NonNull
     @Override
     public SurveyViewHolder onCreateViewHolder(@NonNull ViewGroup viewGroup, int i) {
         arrayCounter++;
-        Log.d("SurveyAdapter", "onCreateViewHolder: optJSON "+surveyArray.optJSONObject(arrayCounter)+" position : "+arrayCounter);
```

Comment:
Recyclerview only keep tabs on the totalnumber of items on screen plus 1, so this will cause issues